chapters/executors.rst

```diff
@@ -0,0 +1,153 @@
+==========================================
+A Guide on What Are Jina Executors and How to Use Them
+==========================================
+
+.. meta::
+   :description: A Guide on What Are Jina Executors and How to Use Them
+   :keywords: Jina, Executors, Drivers
+
+.. note:: This guide assumes you have a basic understanding of Jina, if you haven't, please check out `Jina 101 <https://101.jina.ai>`_ first.
+
+.. contents:: Table of Contents
+    :depth: 2
+
+Motivation
+--------------------
+
+In Jina the Executors represent an algorithmic class and they are a key part to take the full potential of Jina. We have `several of them <https://docs.jina.ai/chapters/all_exec.html`_ and they all live in the `Hub <https://hub.jina.ai/#/home>`_, facilitating the work-flow while creating your app. Here we will see what exactly are they and how you can use them.
```

Comment:
```suggestion
In Jina the Executors represent an algorithmic class and they are a key part to take the full potential of Jina. We have `several of them <https://docs.jina.ai/chapters/all_exec.html>`_ and they all live in the `Hub <https://hub.jina.ai/#/home>`_, facilitating the work-flow while creating your app. Here we will see what exactly are they and how you can use them.
```

Reply:
I don't know why everytime those ticks are randomly changed D: thanks for noticing!